spec/support/fake_analytics.rb

```diff
@@ -80,6 +80,9 @@ def initialize(user: AnonymousUser.new)
   end
 
   def track_event(event, attributes = {})
+    if attributes[:proofing_components].instance_of?(Idv::ProofingComponentsLogging)
+      attributes[:proofing_components] = attributes[:proofing_components].as_json.symbolize_keys
+    end
```

Comment:
I'm going to address this in a follow-up PR. This PR keeps getting out of date with main so I want to get it merged asap

Reply:
Sounds fine to me to follow-up. Maybe we can map values to JSON-ify all of them so that it matches the expected behavior of Ahoy?

```suggestion
    attributes.transform_values!(&:as_json)
```